packages/mdc-side-sheet/README.md

```diff
@@ -1,4 +1,318 @@
+section: components
+excerpt: "Permanent, dismissible, and modal side sheets."
+iconId: side_navigation
+path: /catalog/side-sheets/
+-->
 
-The [Side Sheet component](https://material.io/go/design-sheets-side) is yet to be completed, please follow the [tracking issue](https://github.com/material-components/material-components-web/issues/2665) for more information.
+# Side Sheets
 
+The MDC Side Sheet is used to organize access to destinations and other functionality on an app.
+
+## Design & API Documentation
+
+<ul class="icon-list">
+  <li class="icon-list-item icon-list-item--spec">
+    <a href="https://material.io/go/design-navigation-side-sheet">Material Design guidelines: Navigation side-sheet</a>
+  </li>
+  <li class="icon-list-item icon-list-item--link">
+    <a href="https://material-components.github.io/material-components-web-catalog/#/component/side-sheet">Demo</a>
+  </li>
+</ul>
+
+## Installation
+
+```
+npm install @material/side-sheet
+```
+
+### HTML Structure
+
+```html
+<aside class="mdc-side-sheet">
+  <div class="mdc-side-sheet__content">
+    <div><label><input type="checkbox"> Events</label></div>
+    <div><label><input type="checkbox" checked> Personal</label></div>
+    <div><label><input type="checkbox" checked> Projects</label></div>
+    <div><label><input type="checkbox"> Reminders</label></div>
+    <div><label><input type="checkbox"> Work</label></div>
+  </div>
+</aside>
+```
+
+#### Menu Icon
+
+We recommend using [Material Icons](https://material.io/tools/icons/) from Google Fonts:
+
+```html
+<head>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+</head>
+```
+
+However, you can also use SVG, [Font Awesome](https://fontawesome.com/), or any other icon library you wish.
+
+### Styles
+
+```scss
+@import "@material/side-sheet/mdc-side-sheet";
+```
+
+### JavaScript Instantiation
+
+```js
+import {MDCSideSheet} from "@material/side-sheet";
+const sideSheet = MDCSideSheet.attachTo(document.querySelector('.mdc-side-sheet'));
+```
+
+## Variants
+
+### Dismissible Side Sheet
+
+Dismissible side sheets are by default hidden off screen, and can slide into view. Dismissible side sheets should be used when navigation is not common, and the main app content is prioritized.
+
+```html
+<body>
+  <aside class="mdc-side-sheet mdc-side-sheet--dismissible">
+    <div class="mdc-side-sheet__content">
+      <div><label><input type="checkbox"> Events</label></div>
+      <div><label><input type="checkbox" checked> Personal</label></div>
+      <div><label><input type="checkbox" checked> Projects</label></div>
+      <div><label><input type="checkbox"> Reminders</label></div>
+      <div><label><input type="checkbox"> Work</label></div>
+    </div>
+  </aside>
+
+  <div class="mdc-side-sheet-app-content">
+    App Content
+  </div>
+</body>
+```
+
+> Apply the `mdc-side-sheet-app-content` class to the sibling element after the side-sheet for the open/close animations to work.\
```

Comment:
"the sibling element" -> "a subsequent sibling element" here, and change style to use `~` rather than `+` to make this more agnostic to drawer + side sheet

Reply:
I thought I already replaced the `+` (adjacent sibling selector) with `~` (general sibling selector), but it looks like I missed a few in `mdc-side-sheet-modal.scss`.

Fixed!